### PR TITLE
skip tests - Update readme's again

### PR DESCRIPTION
**Pull request by andy7i**
Ticket: <Link to Jira ticket> (none)
<Put notes here to help reviewer understand this PR>
Readme's were still missing some info
---

A couple follow-on tickets:

Make it easier to run Sam locally and run unit tests https://broadworkbench.atlassian.net/browse/CA-809

Make it easier to run integration tests against a local Sam (or other services) https://broadworkbench.atlassian.net/browse/CA-810

---
**PR checklist**

- [x] I've followed [the instructions](https://github.com/broadinstitute/sam/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [x] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [x] I've updated the [FISMA documentation](https://github.com/broadinstitute/sam/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing

### CODE REVIEW

File: README.md
Comment (gpolumbo-broad) [inline README.md]: I would emphasize that it's the _**template**_ that you're modifying here.  Rewording here slightly to maybe add a sub-heading or a bullut before these instructions to emphasize that it's the template you want to edit.  Or maybe an extra sentence describing how to navigate to the list of templates and find the right ScalaTest one to modify.

File: automation/README.md
Comment (gpolumbo-broad) [inline automation/README.md]: I don't understand this one?  "Local Sam pointing to live dev Sam"?  Do you mean you're running the Sam app locally but using the live dev Sam's database?

File: automation/README.md
Comment (gpolumbo-broad) [inline automation/README.md]: nit: I would suggest that a better way to say this is  to open create a new IntelliJ project with just the automation tests in it.

File: automation/README.md
Comment (gpolumbo-broad) [inline automation/README.md]: These instructions I think are better than the wording in the other README for how to modify the template

**Review by gpolumbo-broad — CHANGES_REQUESTED** [review]
I left a few comments, but I really don't feel a need to block the PR on any of them except the one where I said I didn't understand the meaning of the statement.  

Good clarifications though all around, I like it!

File: automation/README.md
Comment (andy7i) [reply automation/README.md]: I initially was thinking of it as local Sam pointing to the live dev Sam's DB, but it should actually just say a local Sam, because the local Sam can point to either dev DB or a local DB. Thanks for the catch -- updated.

File: automation/README.md
Comment (andy7i) [reply automation/README.md]: I gave it another go. Hopefully my end-of-day brain made some sense🤞

**Review by gpolumbo-broad — APPROVED** [review]

**Comment by andy7i** [comment]
got the ok from doge to merge

**Pull request merged**
